.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,17 @@
+name: pull_request
+
+on:
+  pull_request:
+    branches: [ master ]
```

Comment:
Building PRs targeting master only

Reply:
For now we are developing off of `dev` so we should add that as well.

Reply:
Good catch! I'll update this.

Reply:
Updated [here](https://github.com/covid19risk/covidwatch-ios/pull/20/commits/b11773639a5e75a6fb5381ce5e3cb0f94ccb0649).